projects/mysadna/src/app/app.component.scss

```diff
@@ -1,56 +1,40 @@
-// portrait phones: 576px
-// landscape phones: 768px
-// tablets: 992px
-// desktops: 1200px
-
-.app-container {
-  font-family: 'OpenSans', sans-serif;
-  @media(max-width: 575.98px) {
-    font-size: 10px;
+$logo-size: 74px;
+.app-logo {
+  position: absolute;
+  top: 5px;
+  left: 5px;
+  justify-self: flex-start;
+  display: flex;
+  >.my-sadna-logo {
+    height: $logo-size;
+    width: $logo-size;
```

Comment:
Here's an example of using scss variables.

Reply:
I understand how to use them a little better now, thanks.